external-api/src/v1/controllers/estore/eStoreApi.ts

```diff
@@ -62,47 +64,54 @@ const postToEstore = async (
+  });
+
   return response;
 };
 
 export const createExporterSite = async (exporterName: EstoreSite): Promise<SiteCreationResponse> => {
   const timeout = 1000 * 50; // 50 seconds timeout to handle long timeouts
-  const response = await postToEstore('site', [exporterName], timeout);
+  const response = await postToEstore('sites', [exporterName], timeout);
   return response;
 };
 
 export const addFacilityToTermStore = async (facilityId: EstoreTermStore): Promise<TermStoreResponse> => {
   const timeout = 1000 * 50; // 50 seconds timeout to handle long timeouts
-  const response = await postToEstore(`term/facility`, facilityId, timeout);
+  const response = await postToEstore(`terms/facilities`, [facilityId], timeout);
   return response;
 };
 
-export const createBuyerFolder = async (siteName: string, buyerName: EstoreBuyer): Promise<BuyerFolderResponse> => {
+export const createBuyerFolder = async (siteId: string, buyerName: EstoreBuyer): Promise<BuyerFolderResponse> => {
   const timeout = 1000 * 50; // 50 seconds timeout to handle long timeouts
-  const response = await postToEstore(`site/${siteName}/buyer`, [buyerName], timeout);
+  const response = await postToEstore(`sites/${siteId}/buyers`, [buyerName], timeout);
   return response;
 };
-export const createDealFolder = async (siteName: string, data: EstoreDealFolder): Promise<DealFolderResponse> => {
+export const createDealFolder = async (siteId: string, data: EstoreDealFolder): Promise<DealFolderResponse> => {
   const timeout = 1000 * 120; // 120 seconds timeout to handle long timeouts
-  const response = await postToEstore(`site/${siteName}/deal`, [data], timeout);
+  const response = await postToEstore(`sites/${siteId}/deals`, [data], timeout);
   return response;
 };
-export const createFacilityFolder = async (siteName: string, dealIdentifier: string, data: EstoreFacilityFolder): Promise<FacilityFolderResponse> => {
+export const createFacilityFolder = async (siteId: string, dealIdentifier: string, data: EstoreFacilityFolder): Promise<FacilityFolderResponse> => {
   const timeout = 1000 * 120; // 120 seconds timeout to handle long timeouts
-  const response = await postToEstore(`site/${siteName}/deal/${dealIdentifier}/facility`, [data], timeout);
+  const response = await postToEstore(`sites/${siteId}/deals/${dealIdentifier}/facilities`, [data], timeout);
   return response;
 };
 
 export const uploadSupportingDocuments = async (
-  siteName: string,
+  siteId: string,
   dealIdentifier: string,
   buyerName: string,
   file: EstoreDealFiles,
 ): Promise<UploadDocumentsResponse> => {
   const timeout = 1000 * 50; // 50 seconds timeout to handle long timeouts
-  const response = await postToEstore(`site/${siteName}/deal/${dealIdentifier}/documents?buyerName=${buyerName}`, [file], timeout);
+  const response = await postToEstore(`sites/${siteId}/deals/${dealIdentifier}/documents?buyerName=${buyerName}`, [file], timeout);
```

Comment:
We changed the `buyerName` in this endpoint from a query parameter to a field in the request body to make it more consistent with the other endpoints. We could change it back to a query parameter if that would make it easier for DTFS?

Reply:
good shout. I've updated this now